### PR TITLE
Cocoapods – tvos support

### PR DESCRIPTION
**Pull request by smejkpa7**

### CODE REVIEW

**Comment by matej** [comment]
Is this all it takes to get it working on tvos? #312

**Comment by smejkpa7** [comment]
Yes, I'm already using the forked version in one of my apps.

**Comment by Codeido** [comment]
+1 to approve this pull request

File: MBProgressHUD.podspec [hunk 1]
```diff
@@ -12,7 +12,8 @@ Pod::Spec.new do |s|
   s.license      = { :type => 'MIT', :file => 'LICENSE' }
   s.author       = { 'Matej Bukovinski' => 'matej@bukovinski.com' }
   s.source       = { :git => "https://github.com/matej/MBProgressHUD.git", :tag => s.version.to_s }
-  s.platform     = :ios
+  s.ios.deployment_target	= '8.0'
```
Comment (matej) [inline MBProgressHUD.podspec]: Why do we need to set this. MBProgressHUD works with earlier iOS versions.

File: MBProgressHUD.podspec [hunk 2]
```diff
@@ -12,7 +12,8 @@ Pod::Spec.new do |s|
...
+	s.tvos.deployment_target = '9.0'
```
Comment (matej) [inline MBProgressHUD.podspec]: Inconsistent formating. Tab instead of spaces.

**Comment by smejkpa7** [comment]
Inconsistent formating fixed.
Deployment target for iOS changed to the correct version (4.3).

File: MBProgressHUD.podspec (hunk 2 above)
Comment (smejkpa7) [inline MBProgressHUD.podspec]: updated

File: MBProgressHUD.podspec (hunk 1 above)
Comment (smejkpa7) [inline MBProgressHUD.podspec]: updated

**Comment by matej** [comment]
The static lib is set to `6.0`. Kind of hard to test for even that though. 

Why does it need to be explicitly listed now? Worked without it before for iOS.

**Comment by arsonik** [comment]
As its said in the officila documentation 
https://guides.cocoapods.org/syntax/podspec.html#deployment_target

> New libraries that do require ARC and have a deployment target of
> = iOS 6.0 or OS X >= 10.8:

Because MBProgressHUD does require arc then the correct patch should be :
```
s.ios.deployment_target = '6.0'
s.tvos.deployment_target = '9.0'
```

But there is still an issue :
Pods/MBProgressHUD/MBProgressHUD.m:993:11: 'UIToolbar' is unavailable: not available on tvOS

**Comment by Bo98** [comment]
That documentation is about dispatch_release - something MBProgressHUD does not use. You'll see there is also a bullet point that says:

> New libraries that do require ARC, but have a deployment target of **<** iOS 6.0 or OS X **<** 10.8

That said, Xcode requires iOS 6 for bitcode so it is the minimum that people are adopting.

**Comment by matej** [comment]
Merged #357 which handles this and some other details.

**Pull request closed without merging**